cfn-resources/auditing/resource-role.yaml

```diff
@@ -23,12 +23,7 @@ Resources:
             Statement:
               - Effect: Allow
                 Action:
-                - "secretsmanager:CreateSecret"
```

Comment:
what's the reason why they are unwanted and have to be removed?

Reply:
@marcosuma we are not creating any secrets during any of the Atlas Resource creation now. Earlier we use to save the APIKeys in during APIKey resource creation.However Audit resource doesn't need Secretsmanager.CreateSecret permission at all.